ci-operator/config/redhat-developer/rhdh/.claude/skills/rhdh-aks-tests/SKILL.md

```
@@ -0,0 +1,57 @@
+```bash
+bash "${CLAUDE_SKILL_DIR}/scripts/list-k8s-test-configs.sh" \
+  --pattern "^e2e-aks-"
+```
+
+Filter by branch: add `--branch main`.
+
+## Updating the Version
+
+The K8s version is set per branch as the `MAPT_KUBERNETES_VERSION` env var in each CI config file:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)